back/spec/acceptance/phases_spec.rb

```diff
@@ -153,12 +159,64 @@
         expect(json_response.dig(:data, :relationships, :project, :data, :id)).to eq project_id
       end
 
+      context 'native survey' do
+        let(:phase) { build(:native_survey_phase) }
+        let(:min_budget) { nil }
+        let(:max_budget) { nil }
+
+        example_request 'Create a native survey phase', document: false do
```

Comment:
The `document` option is not supported for `example_request`. `example` should be used instead.

Reply:
Good catch. I always forget. I fixed a few other occurrences.